utam-compiler/src/main/java/utam/compiler/helpers/MethodContext.java

```diff
@@ -65,7 +65,9 @@ public TypeProvider getReturnType(List<ComposeMethodStatement> statements,
     TypeProvider lastStatementReturns = lastStatement.getReturnType();
     if (lastStatementReturns != null
         && methodReturnType != null
-        && !lastStatementReturns.isSameType(methodReturnType)) {
+        && !lastStatementReturns.isSameType(methodReturnType)
+        // Exclude load() method from this check, since it is always VOID type
+        && !methodName.equals("load")) {
```

Comment:
The constant is defined, let's use it, even if we have to make the field `public`.

Reply:
The only reason why I did not do it is because `UtamPageObject` class is not a public class so I cannot access any fields from it, should I make the entire `UtamPageObject `class public?